machineconfiguration/v1/0000_80_kubeletconfig.crd.yaml

```diff
@@ -134,10 +134,6 @@ spec:
                   items:
                     description: KubeletConfigCondition defines the state of the KubeletConfig
                     type: object
-                    required:
-                      - lastTransitionTime
-                      - status
-                      - type
```

Comment:
this is currently not required in the MCO: https://github.com/openshift/machine-config-operator/blob/40571adccaa3ef49a823abe8d7494bcdc48f2021/install/0000_80_machine-config-operator_01_kubeletconfig.crd.yaml#L216C1-L216C1